tests/e2e/MediaCo/portal.spec.js

```diff
@@ -1,6 +1,8 @@
+const path = require('path');
 const {test, expect} = require('@playwright/test');
 const config = require('../../config');
 const common = require('../../common');
+const endpoints = require("../../../sdk-config.json");
```

Comment:
I don't think we should directly access the sdk-config.json this way. Instead, you should be able to import the SdkConfigAccess singleton object and use its API (getSdkConfigServer) to get the same information you use this 'endpoints' variable for.

Reply:
John, Iam able to access the sdk-config.json data through the require statement.

Reply:
Oh, ok. I didn't see that this was the test file. I thought this was in SDK code. In that case, this is ok.